spring-boot-mvc/pom.xml

```diff
@@ -72,6 +72,18 @@
             <version>${spring.fox.version}</version>
         </dependency>
 
+        <!-- ShedLock -->
```

Comment:
It feels weird that this is in `spring-boot-mvc`. Is there a more general one? My concern is this has nothing to do with MVC.